src/main/java/com/spartronics4915/frc2020/subsystems/Indexer.java

```diff
@@ -0,0 +1,71 @@
+package com.spartronics4915.frc2020.subsystems;
+
+import com.spartronics4915.lib.subsystems.SpartronicsSubsystem;
+
+public class Indexer extends SpartronicsSubsystem {
+
+    public int mSpinnerId = -1;
```

Comment:
These should be in Constants.java as `public static final int kSubsystemDescriptorID`.